coordinator/apps/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copy source code and build the project
+FROM node:20-alpine as builder
+
+WORKDIR /builder
+
+COPY . .
+
+RUN npm i -g pnpm@8
+RUN pnpm install --frozen-lockfile --prefer-offline
+RUN pnpm run build
+
+# Create image by copying build artifacts
+FROM node:20-alpine as runner
+RUN npm i -g pnpm@8
+
+RUN mkdir -p ~/rapidsnark/build; \
+    wget -qO ~/rapidsnark/build/prover https://maci-devops-zkeys.s3.ap-northeast-2.amazonaws.com/rapidsnark-linux-amd64-1c137; \
+    chmod +x ~/rapidsnark/build/prover
+RUN wget -qO ~/circom https://github.com/iden3/circom/releases/download/v2.1.6/circom-linux-amd64; \
+    chmod +x ~/circom; \
+    mv ~/circom /bin
```

Comment:
And there are two missing steps:
1. pnpm run download-zkeys:test
2. pnpm run generate-keypair

Reply:
Done!